packages/cli/src/util/get-latest-version/get-latest-worker.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * This file is spawned in the background and checks npm for the latest version
+ * of the CLI, then writes the version to the cache file.
+ *
+ * NOTE: Since this file runs asynchronously in the background, it's possible
+ * for multiple instances of this file to be running at the same time leading
+ * to a race condition where the most recent instance will overwrite the
+ * previous cache file resetting the `notified` flag and cause the update
+ * notification to appear for multiple consequetive commands. Not the end of
+ * the world, but something to be aware of.
+ */
```

Comment:
This comment should also mention that no 3rd part dependencies can be used in this file

Reply:
Fixed